genet/utils/dict_support.py

```diff
@@ -113,6 +113,11 @@ def combine_edge_data_lists(l1, l2):
     return [(u, v, dat) for (u, v), dat in edges.items()]
 
 
+def dict_to_string(d):
+    # turn a (potentially) nested dictionary into a string
```

Comment:
Again - is this comment adding anything? The comment pretty much echoes the function name.